### PR TITLE
Feature/user date filter

### PR DESCRIPTION
**Pull request by MeganHuggins**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling- no new features
- [ ] Test

### Detailed Description
I was able to get the methods responsible for filtering the rooms that are available by the chosen date. I also, have the cards resetting when I new date is picked which then allow the new available rooms to be shown and or hidden accordingly. I got the functionality working for the room type buttons as well. When you click each of them they show their designated room types.
  
### Were there any challenges that arose while implementing this feature? If so, how were the addressed?

!!BUG!! I saw that when I filter it by the available rooms by the chosen date and then try and use the filter by room type buttons it looks like each card it finds is being doubled... not sure what is going on with this or the direction I should go with fixing this. Any ideas would be greatly appreciated!! 

### Files modified:
- [ ] index.html
- [ ] base.scss
- [ ] index.js
- [x] domUpdates.js
- [x] loginHandler.js
- [ ] Hotel.js
- [ ] Users.js
- [ ] recipe-test.js
- [ ] user-test.js
- [ ] Other files:

### CODE REVIEW

File: src/domUpdates.js [hunk 1]
```diff
@@ -109,23 +105,64 @@ const domUpdates = {
...
+
+    roomCards.forEach(card => {
+      const cardType = card.dataset.roomType;
+      if(roomType !== cardType) {
+        card.classList.add('hide')
+      } else {
+        card.classList.remove('hide')
+      }
+    })
+  },
```
Comment (rubyr) [inline src/domUpdates.js:165]: This function could be improved using jQuery:
```javascript
$(".room-card").hide().filter(`[data-room-type="${cardType}"]`).show();
```

File: src/domUpdates.js [hunk 2]
```diff
@@ -26,8 +38,8 @@ const domUpdates = {
 
   loadManagerPortal: (loginInfo, userData, roomData, bookingData) => {
     $('#customer-portal').hide();
-    let customers = mockUserData;
     hotel.sortHotelData(roomData, bookingData);
+    // let customers = userData;
```
Comment (rubyr) [inline src/domUpdates.js:42]: remove commented code before pushing to master

File: src/css/base.scss [hunk 3]
```diff
@@ -336,3 +337,7 @@
 .hide {
   display: none;
 }
+
+.hidden {
+  display: none;
+}
```
Comment (rubyr) [inline src/css/base.scss:343]: two hide classes are unnecessary (if you're relying on one for a certain thing like a filter, try giving it a more semantic name)

File: src/css/base.scss [hunk 4]
```diff
@@ -301,6 +301,7 @@
     text-align: center;
     font-size: 25px;
     text-transform: capitalize;
+    text-transform: lowercase;
```
Comment (rubyr) [inline src/css/base.scss:304]: this line makes the previous line useless

File: src/domUpdates.js (hunk 1 above)
Comment (MeganHuggins) [reply src/domUpdates.js:165]: Okay implemented the above and it worked! Danke

File: src/domUpdates.js (hunk 2 above)
Comment (MeganHuggins) [reply src/domUpdates.js:42]: Deleted

File: src/css/base.scss (hunk 4 above)
Comment (MeganHuggins) [reply src/css/base.scss:304]: Oops! Didn't mean to have that last one in. Deleted

File: src/css/base.scss (hunk 3 above)
Comment (MeganHuggins) [reply src/css/base.scss:343]: Okay took out the hide class

**Pull request merged**